### PR TITLE
Bug 1709640 - add labels for new android jobs

### PR DESCRIPTION
**Pull request by agi**
This change adds some new labels for the jobs being added in Bug 1709640, namely:

- api-16 is being removed from the job name
- new "lite" build configuration for GeckoView

### CODE REVIEW

**Review by sarah-clements — APPROVED** [review]
Looks okay to me, but you'll need to run the lint command locally before this can be merged. `yarn lint --fix` and potentially format per the docs: https://treeherder.readthedocs.io/testing.html#validating-javascript

**Comment by agi** [comment]
Thanks @sarah-clements ! I fixed the linter errors, could you merge this patch when you have time? I don't have permissions.

**Pull request merged**
